tests/func/plots/test_show.py

```diff
@@ -296,3 +296,20 @@ def test_collect_non_existing_dir(tmp_dir, dvc, run_copy_metrics):
     assert "error" in result["workspace"]["data"]["subdir"]
     # make sure others gets loaded
     assert result["workspace"]["data"]["plot.json"]["data"] == metric
+
+
+def test_config_output_dir(tmp_dir, dvc, capsys):
+    subdir = tmp_dir / "subdir"
+    ret = main(["config", "plots.out_dir", os.fspath(subdir)])
+    assert ret == 0
+
+    metric = [{"first_val": 100, "val": 2}, {"first_val": 200, "val": 3}]
+    (tmp_dir / "metric.json").dump_json(metric, sort_keys=True)
+
+    with tmp_dir.chdir():
```

Comment:
Not necessary - we start inside `tmp_dir`. This would be required only if you would be changing the `cwd` previously.